examples/agent_dispatch.py

```diff
@@ -0,0 +1,74 @@
+import asyncio
+import os
+import aiohttp
+from livekit.protocol.room import RoomConfiguration
+from livekit.protocol.agent_dispatch import (
+    RoomAgentDispatch,
+    CreateAgentDispatchRequest,
+)
+from livekit.api import AccessToken, VideoGrants
+from livekit.api.agent_dispatch_service import AgentDispatchService
+
+
+room_name = "my-room"
+agent_name = "test-agent"
+
+"""
+This example demonstrates how to have an agent join a room 
+without using the automatic dispatch. In order to use this 
+feature, you must have an agent running with `agent_name` set 
+when defining your WorkerOptions. A dispatch requests the 
+agent to enter a specific room with optional metadata.
+"""
```

Comment:
link to the docs you wrote too

Reply:
the docs is still not published as DZ is taking it over and creating a dedicated docs page for dispatching. will link to it once that is out